src/Appium.Net/Appium/Android/AndroidDriver.cs

```diff
@@ -211,6 +211,19 @@ public void SetGsmSignalStrength(GsmSignalStrength gsmSignalStrength)
 
         #endregion
 
+        #region Device System
+
+        /// <summary>
+        /// Open the notifications 
+        /// </summary>
+        public void OpenNotifications() => AndroidCommandExecutionHelper.OpenNotifications(this);
+
+        public IDictionary<string, object> GetSystemBars() => AndroidCommandExecutionHelper.GetSystemBars(this);
```

Comment:
it would be useful to have documentation on these methods

Reply:
good spot. done